### PR TITLE
Add Bijaya Pandey in blogs section

### PR DESCRIPTION
**Pull request by bjpandey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,7 @@
 ### Blogs
 - [Amit Chaudhary](http://www.amitness.com)
 - [Ankit Pokhrel](http://ankitpokhrel.com.np/blog)
+- [Bijaya Pandey](http://bijayapandey.com.np/)
```
Comment (geshan) [inline README.md:40]: @bjpandey thanks for the PR, your link looks like a portfolio than a blog. Do you have a blog link?

Comment (bjpandey) [reply README.md:40]: It's okay. I don't have blog link.Thank You @geshan.

Comment (geshan) [reply README.md:40]: Its not ok, the title says `#Blogs` if you have noticed. @techgaun any comments?

Comment (techgaun) [reply README.md:40]: @geshan agreed. The PR should be closed. @bjpandey if you do have a blog, feel free to update the link and re-open the PR.

**Pull request closed without merging**